tests/syslog/test_syslog.py

```diff
@@ -0,0 +1,58 @@
+import time
+import logging
+import pytest
+from common.helpers.assertions import pytest_assert
```

Comment:
After PR #1864 was merged, this form of importing won't work. Please use below importing:
```
from tests.common.helpers.assertions import pytest_assert
```

Reply:
Fixed in the latest commit